examples/platform/silabs/SiWx917/device/src/startup_common_RS1xxxx.c

```diff
@@ -0,0 +1,554 @@
+/**
+ ******************************************************************************
+ * @file      startup_si91x.c
+ * @author    Coocox
+ * @version   V1.0
+ * @date      03/16/2013
```

Comment:
I really doubt that this files is needed in CSA.... Shouldn't it be part of the WiseMCU/WiseConnect or something else ?

Reply:
we have some modification done in Wifi SDK startup files.

That file is not working as it is.